app/services/workflow_errors_reporter.rb

```diff
@@ -4,43 +4,8 @@
 class WorkflowErrorsReporter
 
```

Comment:
Please add a method comment that this will always return true because that is what we ALWAYS get from dor-workflow-service gem and reference https://github.com/sul-dlss/dor-workflow-service/issues/50 in the comment.